Dmytro_Skorobohatskyi/batch_3/rock_paper_scissors.py

```diff
@@ -0,0 +1,243 @@
+import random
+
+possible_weapons = ['rock', 'paper', 'scissors']
+
+def recognize_player_numbers():
+
+    """ Function returns input of player numbers
+
+        Args:
+            (none)
+
+        Returns:
+            int: expected player's number
+
+    """
+    
+    numbers = int(input('Enter amount of players: '))
```

Comment:
What if there will be an exception? You should use isinstance() to validate datatype if it is needed.